index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Godzilla for Hire</title>
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Bowlby+One+SC|Signika" rel="stylesheet">
+</head>
+<body class="body">
+  <header class="header">
+    <nav class="header__nav">
+      <a class="nav__links__current" href="#">Bio</a> | 
+      <a class="nav__links" href="experience.html">Experience/Interests</a>
+    </nav>
+  </header>
+  <main class="main">
+    <h1 class="main__h1">Bio</h1>
+    <p class="main__copy">
+      Godzilla is a professional mutated dinosaur that feeds off nuclear energy. Years ago (millions of years, mind you), Godzilla began life as a Godzillasaurus. At a whopping 12 meters, capable of threatening only minor creatures at that juncture in his life, the then-Godzillasaurus ran into some good and bad luck, hibernating before an awfully large asteroid destroyed most of life on Earth, but waking up later (yes, millions of years—it was a millions-year nap) to nuclear testing. Godzilla tries not to think of himself as a metaphor for post-WWII nuclear proliferation, but he just can't help himself.
+    </p>
+    <div class="full-width">
+      <img class="wide-image" src="assets/godzilla_1.jpg" alt="" />
+    </div>
+    <p class="main__copy">
+      In a following state of animalistic anger and hunger (one might say Godzilla was hangry, and still is), Godzilla earned his reputation as King of the Monsters by both saving Tokyo and other poorly circumstanced cities from King Ghidorah, Battra, Biollante, Mechagodzilla, Destroyah, and monsters of the like, while also managing to destroy those aforementioned poorly circumstanced cities all by himself.
+    </p>
+    <div class="full-width">
+      <img class="wide-image" src="assets/godzilla_2.gif" alt="" />
+    </div>
+    <p class="main__copy">
+      Godzilla is sure to be a great asset on any team bent on destroying the world.
+    </p>
+    <p class="main__copy">
+      To see what Godzilla is up to nowadays, read <a href="https://www.nytimes.com/2014/05/16/movies/godzilla-grandaddy-of-movie-monsters-stomps-back.html?_r=0" alt="Remember, Godzilla is watching you. Probably should click the link... for all our sakes">this review</a> by the <em>New York Times</em> to see what the green, towering, fearsome hype is all about!
```

Comment:
The common convention for external links is to open them up in a new tab/window.  🔨